### PR TITLE
add ip and reverse lookup to amandahosts

### PR DESCRIPTION
**Pull request by mstormi**
Fixes: #1495 

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

File: functions/backup.bash [hunk 1]
```diff
@@ -158,7 +158,10 @@ create_amanda_config() {
   echo -n "$(timestamp) [openHABian] Creating Amanda filesystem... "
   if ! cond_redirect mkdir -p "$configDir" "$databaseDir" "$logDir" "$indexDir"; then echo "FAILED (create directories)"; return 1; fi
   if ! cond_redirect touch "$configDir"/tapelist; then echo "FAILED (touch tapelist)"; return 1; fi
-  if ! (echo -e "${HOSTNAME} ${backupUser}\\n${HOSTNAME} root amindexd amidxtaped\\nlocalhost ${backupUser}\\nlocalhost root amindexd amidxtaped" > "$amandaHosts"); then echo "FAILED (Amanda hosts)"; return 1; fi
+  ip=$(dig +short "$HOSTNAME")
```
Comment (ecdye) [inline functions/backup.bash:161]: Don't we have a helper method to find the IP of a given machine?

Comment (mstormi) [reply functions/backup.bash:161]: no just for the external NAT IP

Comment (ecdye) [reply functions/backup.bash:161]: Ahh, ok.

**Review by ecdye — APPROVED** [review]
LGTM

**Pull request merged**